packages/nextjs/app/builders/0x83DD842BB14aacE72A1a68a129D134Ee3EeBbbd4/page.tsx

```diff
@@ -0,0 +1,45 @@
+import type { NextPage } from "next";
+import { AcademicCapIcon, XMarkIcon } from "@heroicons/react/24/outline";
+import { MapPinIcon } from "@heroicons/react/24/solid";
+
+const Profile: NextPage = () => {
+  return (
+    <>
+      <div className="flex items-center flex-col flex-grow pt-10 justify-center">
+        <div className="flex gap-10">
+          <div className="">
+            <img
```

Comment:
The gh [action fails](https://github.com/BuidlGuidl/batch5.buidlguidl.com/actions/runs/9034520688/job/24828636547?pr=13)  because we are using img tag instead of Next Image tag, 

You could add an eslint ignore comment `eslint-disable-next-line @next/next/no-img-element` above this line so we don't need to configure remote pattern in next.config.js 

or maybe we could add `avatars.githubusercontent.com` to [remote patterns in next config](https://nextjs.org/docs/app/api-reference/components/image#remotepatterns) and use next Image tag here, this way other people can also github avatar for their profile if they want